doc/reference/reference_lua/box_stat/memtx.rst

```diff
@@ -0,0 +1,81 @@
+.. _box_introspection-box_stat_memtx:
+
+box.stat.memtx()
+================
+
+.. module:: box.stat
+
+.. function:: memtx()
+
+    Shows memtx-storage-engine activity.
+
+.. _box_introspection-box_stat_memtx_tx:
+
+box.stat.memtx().tx
+-------------------
+
+* ``tx`` shows the statistics of the memtx transactional manager,
+  which is responsible for transactions (``box.stat.memtx().tx.txn``)
+  and multiversion concurrency control (``box.stat.memtx().tx.mvcc``).
+
+  * ``box.stat.memtx().tx.txn`` shows memory allocation related to transactions.
+
+    It consists of the following sections:
```

Comment:
Not sure if `sections` is the right word here. If it's an object, these may be its _fields_ or something similar.

Reply:
I'd keep "sections" because we refer to them below. "Field" sounds awkward in this context, for example:

```
      For each section, Tarantool reports the following statistics:

      * ``total`` is the number of bytes that are currently allocated in memtx
        for all transactions within the section scope.
```